sleap/gui/commands.py

```diff
@@ -2989,15 +2989,28 @@ def set_visible_nodes(
 
         has_missing_nodes = False
 
+        #calculate scale factor for getting new x and y values
+        old_size_width =  copy_instance.frame.video.shape[2]
+        old_size_height =  copy_instance.frame.video.shape[1]
+        new_size_width =  new_instance.frame.video.shape[2]
+        new_size_height =  new_instance.frame.video.shape[1]
+        scale_width = new_size_width / old_size_width
+        scale_height = new_size_height / old_size_height
```

Comment:
One thing that worries me a bit is if this is always the use case we want. Say for example a user has different sized videos, but the instances are the same size in the videos (i.e. a cropped video). Then, we wouldn't actually want to scale the instance, just offset. But on the other hand, in the issue we are trying to solve the user's instances do scale. 

I suppose we just need to guess which case would be more likely. Scaling is an easier implementation, but I kind of worry that in more cases than not we might actually want to offset the instance if and only if they are out of the frame.

Reply:
Argh I am back and forth. Let's hold-off on merging and changes and bring it up with the group.

Reply:
We decided scaling was best!